packages/yoga/src/Snackbar/web/Snackbar.test.jsx

```diff
@@ -102,4 +109,25 @@ describe('<Snackbar />', () => {
 
     expect(onCloseMock).toHaveBeenCalledTimes(1);
   });
+
+  it('should close with timeout set up', async () => {
```

Comment:
should we start to use 'test' instead 'it' here too?

Reply:
For this project, `it` is the standard